hikari/impl/bot.py

```diff
@@ -319,6 +322,7 @@ def __init__(
         cache_settings: typing.Optional[config_impl.CacheSettings] = None,
         http_settings: typing.Optional[config_impl.HTTPSettings] = None,
         intents: intents_.Intents = intents_.Intents.ALL_UNPRIVILEGED,
+        disable_member_chunks: bool = False,
```

Comment:
I think this would make more sense the other way around (`request_member_chunks: bool = True`) since this would allow for adding an enum which allows for adding more chunking modes down the line if its deemed helpful (where `False` would just be an alias of `MemberChunkingMode.NEVER` and `True` would just be an alias of `MemberChunkingMode.AUTO` for backwards compatibility but for now this is good enough).

Reply:
You're going to have to document the behaviour for it either way. `disable_member_chunks=False` can easily just read like the inverse of `chunk_members=True` or `request_member_chunks=True` (or whatever name you pick) since it kind of is.
None of these names indicate that it will always chunk members went configured to the relevant bool value and you can't protect ppl who don't read the docs